11ty/documentation/examples/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Definition Examples
+parent:
+  title: Documentation
+  href: '/documentation'
+---
+
+Below are links to grab-and-go examples of various common use cases.
+
+- [Basic](basic/): A word. Just a word. There’s nothing wrong with the word. But every words needs a definition.
+- [Flagged Word](flagged-word/): A word. Just a word. There’s nothing wrong with the word. But every words needs a definition.
```

Comment:
For small grammatical/syntax changes, I am going to large-scale ignore them for now until further down the line. But as one call out, this should be "word" not "words.

Reply:
Yip. And the blurb of the flagged item is just a copy of the basic one. 🙈